assignment2.txt

```diff
@@ -0,0 +1,9 @@
+Lloyd Page
+1. Two or more operators can be combined into a single line of code
```

Comment:
This is true - 2+ operators can be combined into a single line of code as long as each operator has the required amount of operands